### ISSUE TITLE
Loading of Control Defaults in the Menu Bar doesnt work

### ISSUE THREAD

**Issue by ehenze** [nsynkde/Tilda#39]

**Comment by jhnnslmk** [comment]
Hi, can you please elaborate?
I tested the persistence of settings quite a bit ... or what are “defaults”?

**Comment by ehenze** [comment]
Yes, it's from the persistence ... Now I set the Defaults on Create inside Tilda to true, bcs taht's currently my desierd behaviour. But before I didn't manage to boot Tilda up with that setting, even through I wrote it into the Settings file.
Also, it might bve something that could be put into a pin on the outside more than from a file, because then it is easier to set the default behaviour differently for different roles( clients).

**Comment by jhnnslmk** [comment]
The pullpush still has an input pin that overrides the UI on Change.

Will add the Send/Receive pins tomorrow to the node and check persistence. The fields internally probably have the correct value and the UI may be initialized with wrong values?

Odd, as I checked it ...
Will let you know about my findings.
